docs/locks.rst

```diff
@@ -72,3 +70,41 @@ multithreaded app.)*
 
     The method raises `tornado.gen.TimeoutError` if there's no notification
     before the deadline.
+
+   Event
+   -----
+   .. autoclass:: Event
+    :members:
+
+    A coroutine can wait for an event to be set. Once it is set, the coroutine
```

Comment:
For the second sentence, how about "Once it is set, calls to ``yield event.wait()`` will not block unless the event has been cleared."